src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -25,7 +26,7 @@ public bool TryGet(string messageId, out OutboxMessage message)
             {
                 session.Advanced.AllowNonAuthoritativeInformation = false;
                 // We use Load operation and not queries to avoid stale results
-                result = session.Load<OutboxRecord>(GetOutboxRecordId(messageId));
+                result = session.Load<OutboxRecord>(new[] { GetOutboxRecordId(messageId), GetOutboxRecordIdWithoutEndpointName(messageId) }).FirstOrDefault(o => o != null);
```

Comment:
`FirstOrDefault()` so can we have both? Or would `SingleOrDefault()` be the better choice?